src/js/components/ItemVolumeTable.js

```diff
@@ -0,0 +1,192 @@
+import classNames from 'classnames';
```

Comment:
I actually called it `ServiceVolumeTable` originally, but Rafa and I decided on the vague `ItemVolumeTable` name because the table can be displayed both on the `TaskDetail` page as well as the `ServiceDetail` page. What do you think?